Module restructure (#141)
kyma-project/btp-manager: module-chart/README.md

</details>

## Overrides

> During rendering by helm a kubernetes resources files, following [values overrides](https://github.com/kyma-project/btp-manager/blob/main/module-chart/overrides.yaml) are appiled.

Reviewer: ```suggestion
While rendering Kubernetes resource files by Helm, the following [values overrides](https://github.com/kyma-project/btp-manager/blob/main/module-chart/overrides.yaml) are applied.
```
